test/schema.test.ts

```diff
@@ -687,6 +689,20 @@ describe('JSON Schema', () => {
       assert.strictEqual(result.items.length, 1);
       assert.strictEqual(result.items[0].label, 'schemastore');
     });
+
+    it('Default snippet with descritpion', async () => {
+      languageSettingsSetup.withSchemaFileMatch({
+        fileMatch: ['test.yaml'],
+        uri: TEST_URI,
+        priority: SchemaPriority.SchemaStore,
+        schema: schemaDefaultSnippetSample,
+      });
+      languageService.configure(languageSettingsSetup.languageSettings);
+      const testTextDocument = setupTextDocument('foo:  ');
+      const result = await languageService.doComplete(testTextDocument, Position.create(0, 5), false);
+      assert.strictEqual(result.items.length, 2);
```

Comment:
Can you do `assert.equal` on the description value here so that we know it's set correctly